README.md

```diff
@@ -2,56 +2,58 @@
 
-1. Meta Update Block: Updates the time in the system, the price (true and oracle) for POKT in USD, simulates the number of transactions on the network, and sets certain timestep tracking variables to 0
-2. Join Block: All logic around servicers, services, gateways, and applications joining.
-3. Stake Block: Any logic for staking that happens in the system.
-4. Delegation and Service Linking Block: Block which takes care of applications delegating to gateways and servicers linking to services.
-5. Relay Request Block: All logic around requests being created and relayed, recording of relays in different groups is also recorded.
-6. Jailing & Slashing Block: Block for servicers or gateways being slashed or jailed as well as the inverse actions + penalties.
-7. Block & Fee Rewards Block: The logic for both the block rewards as well as the fee rewards.
-8. Undelegation & Unservice Block: The block which handles any gateways undelegating or any services being unlinked from.
-9. Leave Block: All logic around servicers, services, gateways, and applications leaving. There is also a substep to check which nodes might be understaked and kick certain ones from the system depending on the policy.
+1. Meta Update Block: updates the time in the system, the price (both true and oracle) for POKT in USD, simulates the number of transactions on the network, and sets certain timestep tracking variables to 0.
+2. Join Block: contains all logic around servicers, services, gateways, and applications joining.
+3. Stake Block: implements the logic for staking in the system.
+4. Delegation and Service Linking Block: takes care of applications delegating to gateways and servicers linking to services.
+5. Relay Request Block: simulates the process of requests being created and relayed, as well as recording of relays in different groups.
+6. Jailing & Slashing Block: implements the steps for servicers or gateways being slashed or jailed, in addition to the inverse actions + penalties.
+7. Block & Fee Rewards Block: logic for both the block rewards as well as the fee rewards.
+8. Undelegation & Unservice Block: handles any gateways undelegating or any services being unlinked from.
+9. Leave Block: all logic around servicers, services, gateways, and applications leaving. There is also a substep to check which nodes might be understaked and kick certain ones from the system depending on the policy.
 
 ## Running Simulations
 
 ### run_experiments
 
-To run pre-packaged simulations, import run_experiments and then pass a list of experiment keys like so:
+To run pre-packaged simulations, import run_experiments and then pass a list of experiment keys. For instance,
 
     from model import run_experiments
-    df = run_experiments(["test1"])
+    df = run_experiments(["test1", "test2])
+
+    would run predefined experiments corresponding to "test1" and "test2". 
```

Comment:
There is no test2 currently, and quotations don't close on it so it would lead to an error when running. There are a few options to pick from other than these such as Base